scout/parse/variant/frequency.py

```diff
@@ -25,37 +25,37 @@ def parse_frequencies(variant, transcripts):
 
     for test_key in thousand_genomes_keys:
         thousand_g = parse_frequency(variant, test_key)
-        if thousand_g:
+        if thousand_g is not None:
```

Comment:
:+1:
Python is sometimes 😈 ..

Reply:
The linters like this syntax :) I have actually learned to write better code thanks to wading through linting hell for a while